syntax				= "proto3";
option optimize_for = CODE_SIZE;

message Cfg_Scene_EnterPoint
{

		uint32 idmap	   = 1;
		uint32 idx		   = 2;
		float  x		   = 3;
		float  y		   = 4;
		float  range	   = 5;
		float  face		   = 6;
		uint32 team_req	   = 10; //组队人数
		uint32 guild_req   = 11; //公会等级
		uint32 lev_req	   = 12; //玩家等级
		uint32 prof_req	   = 13; //玩家职业
		uint32 vip_lev_req = 14; // vip等级
		int32  idphase	   = 15; //位面id,  -1:代表自己的私有位面

}